code/modules/mining/coins.dm

```diff
@@ -106,7 +106,7 @@
 			return
 
 		if(CC.use(1))
-			overlays += image('icons/obj/items.dmi',"coin_string_overlay")
+			overlays += image('icons/obj/economy.dmi',"coin_string_overlay")
```

Comment:
For this to work, I think you actually have to move the sprite as well.
Also, justifying this on the basis of 'immersion' doesn't work very well, as players have no idea what files icons are stored in.

Reply:
That's not the point... the coin sprites were moved here a long time ago, the string sprite included, this function is currently broken, that's what the issue report is about. It's fixing "immersion" by making it so that the coin actually gets the proper string overlay.